config/dlss_baseline.yml

```diff
@@ -84,15 +84,27 @@ Style/BlockComments:
 Style/ClassAndModuleChildren:
   Enabled: false
 
+Style/ClosingParenthesisIndentation:
+  Enabled: false
+
 Style/Documentation:
   Enabled: false
 
+Style/EmptyLinesAroundBlockBody:
+  Enabled: false
+
 Style/EmptyLinesAroundClassBody:
   Enabled: false
 
 Style/EmptyLinesAroundModuleBody:
   Enabled: false
 
+Style/ExtraSpacing:
+  Enabled: false
```

Comment:
I'm concerned that this opens up a lot more extra spacing than just the comment use case that you mention.

Do you feel like the ability to put extra space between in-code comments (which I'm hoping we can minimize the need for as much as possible) outweighs the benefit we would get with being consistent in all the other contexts.  (FWIW, I would be :+1: for adding `AllowForAlignment` since I think it can increase readability, but I don't think that gets at your particular issue).